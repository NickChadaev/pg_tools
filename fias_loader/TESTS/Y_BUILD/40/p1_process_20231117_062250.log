2023-11-17 06:22:50 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_40 postgres /home/n.chadaev@abrr.local/Y_BUILD/40/stage_gar_c_40.csv 2023-11-13
--------------------------------------------------------
[06:22:50] ... Калужская обл., id_region: 39, Execute: /home/n.chadaev@abrr.local/Y_BUILD/40/stage_gar_c_40.csv, process_id:25345
[06:22:50] ... Start process
[06:22:50] ...  -- Version
[06:22:50] ...  -- Очистка от данных
[06:22:50] ...  -- Убираем индексы
[06:22:50] ...  -- Set UNLOGGED
[06:22:50] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[06:22:50] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[06:22:50] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[06:22:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[06:22:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[06:22:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[06:22:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[06:22:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[06:22:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[06:22:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[06:22:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_REESTR_OBJECTS_20231113_27b15f11-27c6-4289-bf5d-62d7e1372acf.XML
[06:33:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_ADDR_OBJ_20231113_cbeaa9aa-b7d1-4e24-946e-ef61dc2f94ef.XML
[06:33:55] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_HOUSES_20231113_4f863e8e-1111-4db9-a6b5-61c9f8170678.XML
[06:44:44] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_STEADS_20231113_5ce21c2e-0623-48be-82dd-9471d4e47f32.XML
[06:45:36] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_APARTMENTS_20231113_31a5888b-1580-4089-bfb3-21515864ea50.XML
[06:50:42] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_ROOMS_20231113_fdd5e477-0cfb-438e-8720-00e492ab1e16.XML
[06:50:44] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_CARPLACES_20231113_6c281770-eecc-45bf-845a-dfff3a1820e5.XML
[06:50:45] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_ADM_HIERARCHY_20231113_4d021a66-8b6e-4385-ba0b-0d606550cb6d.XML
[07:02:31] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_MUN_HIERARCHY_20231113_976e123d-f4ba-403e-b1bb-5498c828cbcd.XML
[07:13:49] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_CHANGE_HISTORY_20231113_2ec30f0e-1c24-4266-bd06-63a005af8fa7.XML
[07:56:16] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_ADDR_OBJ_DIVISION_20231113_64941c4e-823a-46f8-ba30-f345ed3cde43.XML
[07:56:16] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_ADDR_OBJ_PARAMS_20231113_8e49f72f-2d63-4809-b1ca-5546fc87734b.XML
[07:59:41] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_STEADS_PARAMS_20231113_a56fd1e0-4127-4e6e-9871-5eb4573e5205.XML
[08:11:20] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_HOUSES_PARAMS_20231113_461b2f1b-3ef3-4068-8837-f3cb46ae7a2c.XML
[09:26:40] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_APARTMENTS_PARAMS_20231113_a3131b5c-3e54-4e49-bb8b-c72016e93ec8.XML
[09:36:31] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_ROOMS_PARAMS_20231113_bd29bfb4-0938-4785-bdb1-8375515fb8df.XML
[09:36:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_CARPLACES_PARAMS_20231113_34c64994-0107-4e70-80a8-a2f1f848fdf1.XML
[09:36:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/40/AS_NORMATIVE_DOCS_20231113_090dcd8e-f65c-4d31-a655-101e35d3db60.XML
[09:38:23] ...  -- Data finish
[09:38:23] ...  -- Set LOGGED
[09:38:52] ... Stop process
--------------------------------------------------------
