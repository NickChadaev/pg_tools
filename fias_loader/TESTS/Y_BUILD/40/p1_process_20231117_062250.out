 save_gar_version 
------------------
               81
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                81
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 891965
gar_fias.as_addr_obj: 20560
gar_fias.as_houses: 807860
gar_fias.as_steads: 67159
gar_fias.as_apartments: 394316
gar_fias.as_rooms: 2614
gar_fias.as_carplaces: 1371
gar_fias.as_adm_hierarchy: 927236
gar_fias.as_mun_hierarchy: 910414
gar_fias.as_change_history: 3554174
gar_fias.as_addr_obj_division: 76
gar_fias.as_addr_obj_params: 271556
gar_fias.as_steads_params: 916884
gar_fias.as_houses_params: 5989652
gar_fias.as_apartments_params: 730623
gar_fias.as_rooms_params: 2813
gar_fias.as_carplaces_params: 283
gar_fias.as_normative_docs: 135077
